### PR TITLE
Updated views

### PR DESCRIPTION
**Pull request by IuliaUngur**
cc: @asarbu

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,14 +2,7 @@ source 'https://rubygems.org'
...
-# db
-group :development do
-  gem 'sqlite3'
-end
-group :production do
-  gem 'pg'
-end
-
+gem 'pg'
 gem 'pgreset'
```
Comment (asarbu) [inline Gemfile]: why do you need `pgreset` and `pg_search` ?

File: app/assets/javascripts/react_components/live_prediction/react_live_prediction.js.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+var ReactLivePrediction = React.createClass({
+  propTypes: {
+    title: React.PropTypes.string
```
Comment (asarbu) [inline app/assets/javascripts/react_components/live_prediction/react_live_prediction.js.jsx]: why should the title be a props ? I mean on this page can you have a different title ?

File: app/assets/javascripts/react_components/live_prediction/react_live_prediction.js.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  getInitialState: function(){
+    return{
+      title: this.props.title
+    }
+  },
+
+  render: function(){
+    return (
+      <div>
+        <span className="h1">
```
Comment (asarbu) [inline app/assets/javascripts/react_components/live_prediction/react_live_prediction.js.jsx]: why not use `<h1>` instead ?

File: app/views/predictions/live_prediction.haml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+= render 'shared/menu'
+= react_component 'ReactLivePrediction',
```
Comment (asarbu) [inline app/views/predictions/live_prediction.haml]: I think this component is independent of the others and therefor the title does not need to be sent as a prop.

File: app/views/shared/_menu.haml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+%nav.navbar.navbar-default
```
Comment (asarbu) [inline app/views/shared/_menu.haml]: why didn't you try creating the menu as a react component also ? (doing so you will have only react views)

File: app/assets/javascripts/react_components/live_prediction/react_live_prediction.js.jsx (hunk 2 above)
Comment (IuliaUngur) [inline app/assets/javascripts/react_components/live_prediction/react_live_prediction.js.jsx]: meant just as a placeholder to see that it works (definitely not the final form)

Comment (asarbu) [inline app/assets/javascripts/react_components/live_prediction/react_live_prediction.js.jsx]: ok, If you want to use placeholders I think you could write some dummy text as html, or maybe set the initial state to some random text

File: app/views/shared/_menu.haml (hunk 5 above)
Comment (IuliaUngur) [inline app/views/shared/_menu.haml]: didn't want to overly complicate myself. But I'll rewrite it in React

**Pull request merged**